### PR TITLE
Goodies: Remove periods from titles, subtitles and plaintext responses

### PR DESCRIPTION
**Pull request by rasikapohankar**
###### Description of changes

Small change removing periods from titles and subtitles, as pointed out [here](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3086#discussion_r64656831)

###### People to notify (@mention interested parties)

@GuiltyDolphin @moollaza 

---

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/TwelveOclock.pm [hunk 1]
```diff
@@ -40,7 +40,7 @@ handle query => sub {
         # If they guessed, we need to answer the question they asked.
         $guess_result = (!$guess) ? '' : ($guess eq $to_show) ? 'Yes, ' : 'No, ';
     }
-    my $answer = $guess_result . '12:00' . $meridian . ' is ' . $to_show . '.';
+    my $answer = $guess_result . '12:00' . $meridian . ' is ' . $to_show;
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/TwelveOclock.pm]: Can we switch this to `"${guess_result}12:00$meridian is $to_show"`?

Comment (rasikapohankar) [inline lib/DDG/Goodie/TwelveOclock.pm]: @GuiltyDolphin Yes, will change it :+1:

File: lib/DDG/Goodie/TwelveOclock.pm [hunk 2]
```diff
@@ -40,7 +40,8 @@ handle query => sub {
         # If they guessed, we need to answer the question they asked.
         $guess_result = (!$guess) ? '' : ($guess eq $to_show) ? 'Yes, ' : 'No, ';
     }
-    my $answer = $guess_result . '12:00' . $meridian . ' is ' . $to_show;
+
+    my $answer = "${guess_result}12:00$meridian is $to_show";
```
Comment (mintsoft) [inline lib/DDG/Goodie/TwelveOclock.pm]: I think I'd prefer it if you added a space between the `}` and `12` and removed it from the definition of ` $guess_result` on line 41

Comment (rasikapohankar) [inline lib/DDG/Goodie/TwelveOclock.pm]: If we add the space between `guess_result}` and 12:00, the space will get added even when `guess_result` is blank, for a query like `When Is Midnight?`. The space is not noticeable but it's there. Should I still change it?

**Comment by rasikapohankar** [comment]
@GuiltyDolphin Updated the test file for the Cusip IA.

**Comment by GuiltyDolphin** [comment]
@rasikapohankar Thanks for this - I designate thee as the Period Police. :smile:

**Comment by rasikapohankar** [comment]
Thanks @GuiltyDolphin! #3570 can be closed too :+1:

**Pull request merged**